Sources/Twift+Lists.swift

```diff
@@ -103,6 +103,26 @@ extension Twift {
   public func deleteList(_ listId: List.ID) async throws -> TwitterAPIData<DeleteResponse> {
     return try await call(route: .list(listId), method: .DELETE, expectedReturnType: TwitterAPIData.self)
   }
+    
+    /// Enables the authenticated user to create a new List.
+    /// - Parameters:
+    ///  - name: List name (required)
+    ///  - description: Description for the list (optional)
+    ///  - private: true or false (optional)
+    /// - Returns: A response object containing the name and the ID of the list.
+    public func createList(name: String, description: String? = "", isPrivate: Bool? = false) async throws -> TwitterAPIData<CreatedListResponse> {
+        
+        let body: [String : Any] = [
+            "name": name,
+            "description": description ?? "",
+            "private": isPrivate ?? false
```

Comment:
Since we already have default values for `description` and `isPrivate`, you don't need to do this nullish coalescing. Just make the types for the arguments non-optional.

Reply:
On second thought, making `description` optional does make sense since